### PR TITLE
eGrants - Add and remove flag maintenance for a grant year

### PR DESCRIPTION
**Pull request by hamidnesar**
These are old requirements hence there are no Story Ids associated to them.

### CODE REVIEW

**Comment by hamidnesar** [comment]
Reviewed

**Review by Mariachaudhry — APPROVED** [review]

**Review by Mariachaudhry — APPROVED** [review]

File: src/test/java/CustomBusiness/Egrants/Steps/QuickLinkVerificationAndManagementMenuSteps.java
Comment (iamdez99) [inline src/test/java/CustomBusiness/Egrants/Steps/QuickLinkVerificationAndManagementMenuSteps.java:1]: You have a 3 second hard sleep before handling an alert. This is okay for now but we should create a common method later to wait for an alert to be present.

File: src/test/java/CustomBusiness/Egrants/StepsImplementation/EgrantsStepImplementation.java
Comment (iamdez99) [inline src/test/java/CustomBusiness/Egrants/StepsImplementation/EgrantsStepImplementation.java:1]: Can you please use the SoftAssert from the Hooks class?

**Review by iamdez99 — CHANGES_REQUESTED** [review]
Please see comments.

File: src/test/java/CustomBusiness/Egrants/Steps/QuickLinkVerificationAndManagementMenuSteps.java
Comment (hamidnesar) [reply src/test/java/CustomBusiness/Egrants/Steps/QuickLinkVerificationAndManagementMenuSteps.java:1]: I will make sure to create a common method in my next PR

File: src/test/java/CustomBusiness/Egrants/StepsImplementation/EgrantsStepImplementation.java
Comment (hamidnesar) [reply src/test/java/CustomBusiness/Egrants/StepsImplementation/EgrantsStepImplementation.java:1]: Replaced the hard asserts with SoftAssert

Comment (iamdez99) [inline src/test/java/CustomBusiness/Egrants/StepsImplementation/EgrantsStepImplementation.java:1]: Please remove line 12

**Review by iamdez99 — CHANGES_REQUESTED** [review]

**Review by iamdez99 — CHANGES_REQUESTED** [review]
Please view comments

**Review by iamdez99 — APPROVED** [review]

**Pull request merged**